### PR TITLE
There is no “sha256sum” on Mac OS X whereas it works fine with “shasum”

### PR DESCRIPTION
**Pull request by loxal**
eventually add an OS switch, maybe something like if “$HOME starts with “/Users/"”…

### CODE REVIEW

**Comment by ebfull** [comment]
`shasum` defaults to SHA1, so you'll need to do `shasum -a 256`. Anyway, can we check if `sha256sum` is available and use that, otherwise use `sha256sum -a 256`? That should be sufficient for this script.

**Comment by loxal** [comment]
sha256sum is not available and I also could not find it in any brew package (Mac’s apt-get). However *shasum -a 256* would work.

**Comment by ebfull** [comment]
I mean, make the bash script check if `sha256sum` is available first before it tries to resort to `shasum -a 256`.

**Comment by daira** [comment]
Are there likely to be platforms with ``sha256sum`` but no ``shasum -a 256``? If not then let's just use the latter.

File: zcutil/fetch-params.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ cd "$PARAMS_DIR"
 
 # Now verify their hashes:
 echo 'Verifying parameter file integrity via sha256sum...'
-sha256sum --check - <<EOF
+shasum --check - <<EOF
```
Comment (defuse) [inline zcutil/fetch-params.sh]: This should be `shasum -a 256 --check << EOF`

**Comment by ebfull** [comment]
@loxal Can you rebase this onto latest and change it to `shasum -a 256 --check << EOF`? If not, we can do it. :) Thanks.

**Comment by loxal** [comment]
@ebfull I just updated my PR

**Comment by ebfull** [comment]
This has to be rebased on top of `zc.v0.11.2.latest` because other parts of fetch-params were changed.

**Comment by loxal** [comment]
@ebfull ok, let‘s try one more time: I’ve committed the changes now on top of a clean `zc.v0.11.2.latest` branch, pulled from `origin/zcash`.

**Comment by ebfull** [comment]
You may not be updating the correct branch, or perhaps github is buggy. This is the commit I see on your `patch-1` branch: https://github.com/loxal/zcash/commit/3c02b3be80b6c88154586aeab8a4fc08c0dd86e0

**Comment by loxal** [comment]
The patch name stays the same but the contents should be on top of your current main branch, `zc.v0.11.2.latest`.

**Comment by loxal** [comment]
Try this one https://github.com/zcash/zcash/pull/955. I’ve created a new PR with the same content.

**Comment by JohannElmo** [comment]
great job - thanks

**Pull request closed without merging**